### DISCUSSION TITLE
Actividad 0  - Sbert Tomas

### DISCUSSION THREAD

**Discussion by TomaSbert** [frlp-utn-ingsoft/catedra#23] [Show and tell]
https://github.com/TomaSbert/IngSoftware

**Comment by RodrigoJacznik** [comment]
Buenas.
Los mensajes de commits estan bien.
No veo la parte 4 del tutorial (https://docs.djangoproject.com/en/5.0/intro/tutorial04/) donde se crea el formulario para votar.
Por favor corregilo para aprobar la actividad.
Saludos

**Comment by TomaSbert** [comment]
Hola Rodrigo, me explicas qué es lo que no podes ver? Porque a medida que fui haciendo la parte 4 esta commiteada, esta como 4.1 / 4.2 / 4.3.
Tal cual lo decia el tutorial


El 11 abr 2024, a la(s) 23:22, Rodrigo Jacznik ***@***.***> escribió:

﻿

Buenas.
Los mensajes de commits estan bien.
No veo la parte 4 del tutorial (https://docs.djangoproject.com/en/5.0/intro/tutorial04/) donde se crea el formulario para votar.
Por favor corregilo para aprobar la actividad.
Saludos

**Comment by TomaSbert** [comment]
Creo que ya commitie lo que me faltaba, lo podes encontrar en el git como [4.0] cualquier cosa avíseme profe!

El 12 abr 2024, a la(s) 08:38, tomas sbert ***@***.***> escribió:

﻿

Hola Rodrigo, me explicas qué es lo que no podes ver? Porque a medida que fui haciendo la parte 4 esta commiteada, esta como 4.1 / 4.2 / 4.3.
Tal cual lo decia el tutorial


El 11 abr 2024, a la(s) 23:22, Rodrigo Jacznik ***@***.***> escribió:

﻿

Buenas.
Los mensajes de commits estan bien.
No veo la parte 4 del tutorial (https://docs.djangoproject.com/en/5.0/intro/tutorial04/) donde se crea el formulario para votar.
Por favor corregilo para aprobar la actividad.
Saludos

**Comment by RodrigoJacznik** [comment]
Si, ahora funciona. 
Se puede refactorizar el archivo https://github.com/TomaSbert/IngSoftware/blob/main/mysite/polls/views.py donde tenes varias funciones con el mismo nombre.
En la db tampoco veo `choice` dentro de la `poll` creada. Como probaste la vista de `results`?

**Comment by TomaSbert** [comment]
Perfecto que ya funcione, te consulto las funciones que decís, serían estas verdad? Las elimino?
[image0.jpeg]

El 12 abr 2024, a la(s) 11:36, Rodrigo Jacznik ***@***.***> escribió:

﻿

Si, ahora funciona.
Se puede refactorizar el archivo https://github.com/TomaSbert/IngSoftware/blob/main/mysite/polls/views.py donde tenes varias funciones con el mismo nombre

**Reply by RodrigoJacznik** [reply]
no se subió la captura.

**Comment by TomaSbert** [comment]
[image0.jpeg]

**Reply by RodrigoJacznik** [reply]
antes de subir un comentario revisa la pestaña de `preview` que te muestra como va a quedar tu mensaje.
![image](https://github.com/frlp-utn-ingsoft/catedra/assets/2715743/2f5665bd-2f95-45d9-8ede-5dd3dbc1cead)

**Comment by TomaSbert** [comment]
Perdon profe, te estaba contestando por el mail.
Al parecer via mail, no se cargan las fotos. Ahora inicie en la pc el git y te contesto por aca.
![WhatsApp Image 2024-04-12 at 11 51 07](https://github.com/frlp-utn-ingsoft/catedra/assets/133404057/c4839cce-ce6b-4083-a887-50a57be6bed6)

**Comment by RodrigoJacznik** [comment]
Tenes que borrar las siguientes funciones:

https://github.com/TomaSbert/IngSoftware/blob/89e1ee04d359a758cba5b1b694d8aa7a06cfb671/mysite/polls/views.py#L29
https://github.com/TomaSbert/IngSoftware/blob/89e1ee04d359a758cba5b1b694d8aa7a06cfb671/mysite/polls/views.py#L32
https://github.com/TomaSbert/IngSoftware/blob/89e1ee04d359a758cba5b1b694d8aa7a06cfb671/mysite/polls/views.py#L39
https://github.com/TomaSbert/IngSoftware/blob/89e1ee04d359a758cba5b1b694d8aa7a06cfb671/mysite/polls/views.py#L66

**Reply by TomaSbert** [reply]
Muchas gracias profe, no me había dado cuenta.
Los cambios lo subí al git esta como Parte 4.0.1  ->  https://github.com/TomaSbert/IngSoftware/commits/main/

**Comment by RodrigoJacznik** [comment]
Genial.
La app funciona bien. Tuve que agregar algunas `choice` a la db para poder probar la app.
La actividad esta aprobada.
